### PR TITLE
Frontend samata

### PR DESCRIPTION
**Pull request by sammu1991**
I connected mine with luna's.. navigation is working

### CODE REVIEW

File: roomranger/frontend/src/App.css [hunk 1]
```diff
@@ -1,4 +1,31 @@
...
+ color: white;
+}
+
+.nbar{
+  background-color: darkblue;
+}
+
+
+
+/* .App {
```
Comment (LunaLiu829) [inline roomranger/frontend/src/App.css:28]: I think you command this that's why the buttons are not in the center.

**Comment by sammu1991** [comment]
If I keep as it is my headings are getting cut..

**Comment by sammu1991** [comment]
Did you try without it? I haven’t used “header” anywhere. Also i checked
taking it off.. it’s still the same.

File: roomranger/src/main/java/org/launchcode/roomranger/data/RoomAssignedRepository.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package org.launchcode.roomranger.data;
```
Comment (LunaLiu829) [inline roomranger/src/main/java/org/launchcode/roomranger/data/RoomAssignedRepository.java]: I think you can delete all the related to assigned room data, I think Francesca will go to build them.

File: roomranger/src/main/java/org/launchcode/roomranger/models/DTO/RegistrationFormDTO.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
+package org.launchcode.roomranger.models.DTO;
```
Comment (LunaLiu829) [inline roomranger/src/main/java/org/launchcode/roomranger/models/DTO/RegistrationFormDTO.java]: You created this DTO which I think you will not use it anymore.

File: roomranger/src/main/java/org/launchcode/roomranger/models/RoomAssigned.java [hunk 4]
```diff
@@ -0,0 +1,121 @@
+package org.launchcode.roomranger.models;
```
Comment (LunaLiu829) [inline roomranger/src/main/java/org/launchcode/roomranger/models/RoomAssigned.java]: Delete this one I think

File: roomranger/src/main/java/org/launchcode/roomranger/models/RoomAttendantProfile.java [hunk 5]
```diff
@@ -1,5 +1,6 @@
 package org.launchcode.roomranger.models;
 
-public class RoomAttendantProfile {
+public class RoomAttendantProfile extends RoomAttendant {
```
Comment (LunaLiu829) [inline roomranger/src/main/java/org/launchcode/roomranger/models/RoomAttendantProfile.java]: Delete it.

File: roomranger/src/main/java/org/launchcode/roomranger/models/Task.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
+package org.launchcode.roomranger.models;
```
Comment (LunaLiu829) [inline roomranger/src/main/java/org/launchcode/roomranger/models/Task.java]: This one Francesca will create it I think.

File: roomranger/src/main/java/org/launchcode/roomranger/controllers/RegistrationController.java [hunk 7]
```diff
@@ -0,0 +1,120 @@
+package org.launchcode.roomranger.controllers;
```
Comment (LunaLiu829) [inline roomranger/src/main/java/org/launchcode/roomranger/controllers/RegistrationController.java]: Sonal working on the registration. Please delete this one.

**Comment by sammu1991** [comment]
Resolved all the mentioned ones.

**Pull request merged**